<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkProxySettingsPage</class>
 <widget class="QWidget" name="NetworkProxySettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Proxy</string>
  </property>
  <layout class="QVBoxLayout" name="layout_networkproxysettingspage">
   <item>
    <widget class="QRadioButton" name="proxy_system">
     <property name="text">
      <string>&amp;Use the system proxy settings</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="proxy_direct">
     <property name="text">
      <string>Direct internet connection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="proxy_manual">
     <property name="text">
      <string>&amp;Manual proxy configuration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="proxy_manual_container" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="layout_proxy_manual_container">
      <property name="leftMargin">
       <number>24</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="layout_proxy_selection">
        <item>
         <widget class="QComboBox" name="proxy_type">
          <item>
           <property name="text">
            <string>HTTP proxy</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SOCKS proxy</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="proxy_hostname">
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_port">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="proxy_port">
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>8080</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="proxy_auth">
        <property name="title">
         <string>Use authentication</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QFormLayout" name="formLayout_7">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_username">
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="proxy_username">
           <property name="text">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_password">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="proxy_password">
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="proxy_engine">
        <property name="text">
         <string>Use proxy settings for streaming</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer_bottom">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>proxy_system</tabstop>
  <tabstop>proxy_direct</tabstop>
  <tabstop>proxy_manual</tabstop>
  <tabstop>proxy_type</tabstop>
  <tabstop>proxy_hostname</tabstop>
  <tabstop>proxy_port</tabstop>
  <tabstop>proxy_auth</tabstop>
  <tabstop>proxy_username</tabstop>
  <tabstop>proxy_password</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>proxy_manual</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxy_manual_container</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
